Applications/LLMEval/ContentView.swift

```diff
@@ -32,6 +43,20 @@ struct ContentView: View {
             }
         }
         .padding()
+        .toolbar {
+            ToolbarItem(placement: .primaryAction) {
+                Button {
+                    Task {
+                        copyToClipboard(llm.output)
+                    }
+                } label: {
+                    Label("Copy Output", systemImage: "doc.on.doc.fill")
+                }
+                .disabled(llm.output == "")
+                .labelStyle(.titleAndIcon)
```

Comment:
Curious, why add a toolbar item?  The text selection is enabled so the user can just select whatever they want and copy it normally.

Reply:
I added that toolbar item to copy all of text output at once for two reasons:

1. Markdown block appends new content as blocks (header, code etc.). This results in a text selection that is block wise. At the moment, I don't know of a elegant way to solve the issue of selecting all the text from the output when using Markdown rendering. 
2. When the total token count is higher, I believe it would be handy to have a button to copy all the output at once.